src/server.rs

```diff
@@ -217,9 +217,14 @@ impl Server {
             None => info!(target: "lazymc", "Starting server..."),
         }
 
+        if config.server.freeze_process {
```

Comment:
Future idea: if this fails, we may fall back to shutting down instead.

You can leave this as is for now. :+1: